roles/kubernetes/preinstall/tasks/0040-set_facts.yml

```diff
@@ -176,3 +176,10 @@
   set_fact:
     kubelet_flexvolumes_plugins_dir: /var/lib/kubelet/volumeplugins
   when: not usr.stat.writeable
+
+- name: Ensure IPv6DualStack featureGate is set when enable_dual_stack_networks is true
+  set_fact:
+    kube_feature_gates: "{{ kube_feature_gates + [ 'IPv6DualStack=true' ] }}"
+  when:
+    - enable_dual_stack_networks
+    - "{{ 'IPv6DualStack=true' not in kube_feature_gates }}"
```

Comment:
`- 'IPv6DualStack=true' not in kube_feature_gates` ?

Reply:
Original commit updated

Reply:
It's not happy with that format:
* https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/985727700

Reply:
`- not 'IPv6DualStack=true' in kube_feature_gates` maybe ?

Reply:
Looks happier now, thanks :)